app/api.py

```diff
@@ -580,6 +625,60 @@ def channel_acl(self, id, channel_id):
         data = obj_to_dict(server.getACL(channel_id))
         return Response(json.dumps(data, sort_keys=True, indent=4), mimetype='application/json')
 
+    @conditional(auth.login_required, auth_enabled)
+    @route('<int:id>/channels/<int:channel_id>/acl', methods=['PATCH'])
+    def update_channel_acl(self, id, channel_id):
+        """ Update specific channel ACL
+        """
+
+        server = meta.getServer(id)
+
+        # Return 404 if not found
+        if server is None:
+            return jsonify(message="Not Found"), 404
+
+        origin_acl = server.getACL(channel_id)
+        update_acls = origin_acl[0]
+        update_groups = origin_acl[1]
+        update_inherit = origin_acl[2]
+
+        params = request.get_json()
```

Comment:
I see you're reading the request JSON body here. However, all of the methods read via multipart form data, so I would rather not change it if possible.

Looking back at the project, I do wish I went with reading the JSON body instead though. 🙂

Reply:
It is ok.
I just think json is easy to pass array data like `acls` or `groups`.

Can back to use form data.

Reply:
here form data with array, you have any idea?